internal/services/fabric/fabric_capacity_resource_test.go

```diff
@@ -0,0 +1,207 @@
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func TestAccFabricCapacity_requiresImport(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_fabric_capacity", "test")
+	r := FabricCapacityResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
+
+func TestAccFabricCapacity_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_fabric_capacity", "test")
+	r := FabricCapacityResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func TestAccFabricCapacity_update(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_fabric_capacity", "test")
+	r := FabricCapacityResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.update(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func (r FabricCapacityResource) Exists(ctx context.Context, clients *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := fabriccapacities.ParseCapacityID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := clients.Fabric.FabricCapacitiesClient.Get(ctx, *id)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving %s: %+v", *id, err)
+	}
+	return pointer.To(resp.Model != nil), nil
+}
+
+func (r FabricCapacityResource) template(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctest-rg-%d"
+  location = "%s"
+}
+`, data.RandomInteger, data.Locations.Primary)
+}
+
+func (r FabricCapacityResource) basic(data acceptance.TestData) string {
+	template := r.template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_fabric_capacity" "test" {
+  name                   = "acctestffc%d"
+  resource_group_name    = azurerm_resource_group.test.name
+  location               = "%s"
+  administration_members = [data.azurerm_client_config.current.object_id]
+
+  sku {
+    name = "F32"
```

Comment:
Also why aren't we using a cheaper sku here like F2? 

F32 can cost up to 5139 USD a month if the test fails or there's a problem with deletion.

Reply:
Fixed.